packages/block-library/src/tag-cloud/edit.js

```diff
@@ -68,15 +63,14 @@ function TagCloudEdit( { attributes, setAttributes, taxonomies } ) {
 			value: '',
 			disabled: true,
 		};
-		const taxonomyOptions = map(
-			taxonomies?.filter( ( tax ) => !! tax.show_cloud ),
-			( item ) => {
+		const taxonomyOptions = ( taxonomies ?? [] )
```

Comment:
This actually fixes a bug introduced in #46114. 

Optionally chaining `.filter()` here will result in making `taxonomyOptions` to be `undefined` if there are no taxonomies registered. This will work with the original `_.map()` since it will just return an empty array, but not with `Array.prototype.map()`. 

It's a subtle bug since 99.99% of the WordPress installations will contain at least one registered taxonomy, but still, this is fixing it for the remaining 0.001%.